tests/third_party/cupy/fft_tests/test_fft.py

```diff
@@ -324,7 +324,6 @@ def test_ihfft(self, xp, dtype):
     {"n": 10, "d": 0.5},
     {"n": 100, "d": 2},
 )
```

Comment:
Should we update third party tests and to align with the latest changes there, something like:
```python
@pytest.fixture
def skip_forward_backward(request):
    if request.instance.norm in ("backward", "forward"):
        if not (np.lib.NumpyVersion(np.__version__) >= "1.20.0"):
            pytest.skip("forward/backward is supported by NumPy 1.20+")

# ...

@pytest.mark.usefixtures("skip_forward_backward")
@testing.parameterize(
    *testing.product(
        {
            "n": [None, 0, 5, 10, 15],
            "shape": [(0,), (10, 0), (10,), (10, 10)],
            "norm": [None, "backward", "ortho", "forward", ""],
        }
    )
)
class TestFft:
    @testing.for_all_dtypes()
    @testing.numpy_cupy_allclose(
        rtol=1e-4,
        atol=1e-7,
        accept_error=ValueError,
        contiguous_check=False,
        type_check=has_support_aspect64(),
    )
    def test_fft(self, xp, dtype):
        a = testing.shaped_random(self.shape, xp, dtype)
        out = xp.fft.fft(a, n=self.n, norm=self.norm)

        # np.fft.fft always returns np.complex128
        if xp is np and dtype in [np.float16, np.float32, np.complex64]:
            out = out.astype(np.complex64)

        return out

    @testing.for_all_dtypes()
    @testing.numpy_cupy_allclose(
        rtol=1e-4,
        atol=1e-7,
        accept_error=ValueError,
        contiguous_check=False,
        type_check=has_support_aspect64(),
    )
    # NumPy 1.17.0 and 1.17.1 raises ZeroDivisonError due to a bug
    @testing.with_requires("numpy!=1.17.0")
    @testing.with_requires("numpy!=1.17.1")
    def test_ifft(self, xp, dtype):
        a = testing.shaped_random(self.shape, xp, dtype)
        out = xp.fft.ifft(a, n=self.n, norm=self.norm)

        if xp is np and dtype in [np.float16, np.float32, np.complex64]:
            out = out.astype(np.complex64)

        return out


@testing.parameterize(
    *testing.product(
        {
            "shape": [(0, 10), (10, 0, 10), (10, 10), (10, 5, 10)],
            "data_order": ["F", "C"],
            "axis": [0, 1, -1],
        }
    )
)
class TestFftOrder:
    @testing.for_all_dtypes()
    @testing.numpy_cupy_allclose(
        rtol=1e-4,
        atol=1e-7,
        accept_error=ValueError,
        contiguous_check=False,
        type_check=has_support_aspect64(),
    )
    def test_fft(self, xp, dtype):
        a = testing.shaped_random(self.shape, xp, dtype)
        if self.data_order == "F":
            a = xp.asfortranarray(a)
        out = xp.fft.fft(a, axis=self.axis)

        # np.fft.fft always returns np.complex128
        if xp is np and dtype in [np.float16, np.float32, np.complex64]:
            out = out.astype(np.complex64)

        return out

    @testing.for_all_dtypes()
    @testing.numpy_cupy_allclose(
        rtol=1e-4,
        atol=1e-7,
        accept_error=ValueError,
        contiguous_check=False,
        type_check=has_support_aspect64(),
    )
    def test_ifft(self, xp, dtype):
        a = testing.shaped_random(self.shape, xp, dtype)
        if self.data_order == "F":
            a = xp.asfortranarray(a)
        out = xp.fft.ifft(a, axis=self.axis)

        if xp is np and dtype in [np.float16, np.float32, np.complex64]:
            out = out.astype(np.complex64)

        return out
```